documentation/modules/cruise-control/proc-generating-rebalance-proposals.adoc

```diff
@@ -0,0 +1,98 @@
+// Module included in the following assemblies:
+//
+// assembly-cruise-control-concepts.adoc
+
+[id='proc-generating-rebalance-proposals-{context}']
+= Generating rebalance proposals 
+
+When you create or update a `KafkaClusterRebalance` resource, Cruise Control generates a _rebalance proposal_ in JSON.
+This summarizes an optimal mapping of partitions to brokers for the Kafka cluster based on the rebalance goals that were used along with statistics on the cluster rebalance.
```

Comment:
So it is not just partitions to brokers, it also proposes changes to partition replica leaders.

Reply:
Reformatted as a bulleted list.